benchmarks/src/main/scala/bloop/logging/NoOpLogger.scala

```diff
@@ -0,0 +1,13 @@
+package bloop.logging
+
+object NoOpLogger extends AbstractLogger {
+  override def name: String = "NoOpLogger"
+  override def ansiCodesSupported(): Boolean = true
+
+  override def debug(msg: String): Unit = ()
+  override def error(msg: String): Unit = ()
+  override def warn(msg: String): Unit = ()
+  override def trace(exception: Throwable): Unit = ()
+  override def info(msg: String): Unit = ()
+  override def verbose[T](op: => T): T = op
```

Comment:
Are we sure we need this logger at all? What's the value of benchmarking logging?

Reply:
It's more about *not* benchmarking logging when we want to benchmark other things.